aiida/cmdline/commands/cmd_status.py

```diff
@@ -14,13 +14,26 @@
 
 import sys
 
+import logging
 from enum import IntEnum
+from contextlib import contextmanager
 import click
 
 from aiida.cmdline.commands.cmd_verdi import verdi
 from ..utils.echo import ExitCode
 
 
+@contextmanager
+def set_log_level(logger, level):
+    'Temporarily adjust the log-level of logger.'
```

Comment:
For docstrings we try to follow the conventions of [PEP257](https://www.python.org/dev/peps/pep-0257/). So in this case the single quotes should be replaced with triple double quotes, i.e. `"""Temporarily adjust the log-level of logger."""`

Reply:
Got it. This was automatically changed from double to single quotes by the pre-commit hook.